sdk/identity/azure-identity/tests/recordings/test_imds_credential.pyTestImdstest_system_assigned.json

```diff
@@ -0,0 +1,56 @@
+      "RequestMethod": "GET",
+      "RequestHeaders": {
+        "Accept": "*/*",
+        "Accept-Encoding": "gzip, deflate",
+        "Connection": "keep-alive",
+        "User-Agent": "azsdk-python-identity/1.8.0 Python/3.9.9 (Windows-10-10.0.19042-SP0)"
+      },
+      "RequestBody": null,
+      "StatusCode": 400,
+      "ResponseHeaders": {
+        "Content-Length": "88",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Wed, 26 Jan 2022 19:47:30 GMT",
+        "Server": "IMDS/150.870.65.528"
+      },
+      "ResponseBody": {
+        "error": "invalid_request",
+        "error_description": "Required metadata header not specified"
+      }
+    },
+    {
+      "RequestUri": "http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01\u0026resource=https://management.azure.com",
+      "RequestMethod": "GET",
+      "RequestHeaders": {
+        "Accept": "*/*",
+        "Accept-Encoding": "gzip, deflate",
+        "Connection": "keep-alive",
+        "Metadata": "true",
+        "User-Agent": "azsdk-python-identity/1.8.0 Python/3.9.9 (Windows-10-10.0.19042-SP0)"
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "Content-Length": "1718",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Wed, 26 Jan 2022 19:47:30 GMT",
+        "Server": "IMDS/150.870.65.528"
+      },
+      "ResponseBody": {
+        "access_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik1yNS1BVWliZkJpaTdOZDFqQmViYXhib1hXMCIsImtpZCI6Ik1yNS1BVWliZkJpaTdOZDFqQmViYXhib1hXMCJ9.eyJhdWQiOiJodHRwczovL21hbmFnZW1lbnQuYXp1cmUuY29tIiwiaXNzIjoiaHR0cHM6Ly9zdHMud2luZG93cy5uZXQvNzJmOTg4YmYtODZmMS00MWFmLTkxYWItMmQ3Y2QwMTFkYjQ3LyIsImlhdCI6MTY0MzIyNjA2MSwibmJmIjoxNjQzMjI2MDYxLCJleHAiOjE2NDMzMTI3NjEsImFpbyI6IkUyWmdZSmhUR1A0aUt1Wi9CT1BVbzQ5NDMrektCd0E9IiwiYXBwaWQiOiJmZTgzODY1Yi00OTVlLTRlYzctOTU3NC0xMmIyNjQ4ODJjOWYiLCJhcHBpZGFjciI6IjIiLCJpZHAiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC83MmY5ODhiZi04NmYxLTQxYWYtOTFhYi0yZDdjZDAxMWRiNDcvIiwiaWR0eXAiOiJhcHAiLCJvaWQiOiI0NTZjMjQ3OS0yOTJjLTQzNzAtOTFiMS1lNGVhMzBjYTUxMzIiLCJyaCI6IjAuQVJvQXY0ajVjdkdHcjBHUnF5MTgwQkhiUjBaSWYza0F1dGRQdWtQYXdmajJNQk1hQUFBLiIsInN1YiI6IjQ1NmMyNDc5LTI5MmMtNDM3MC05MWIxLWU0ZWEzMGNhNTEzMiIsInRpZCI6IjcyZjk4OGJmLTg2ZjEtNDFhZi05MWFiLTJkN2NkMDExZGI0NyIsInV0aSI6IlpnalI0bkEyS1VPMG15SFcwNlhDQUEiLCJ2ZXIiOiIxLjAiLCJ4bXNfbWlyaWQiOiIvc3Vic2NyaXB0aW9ucy8yY2Q2MTdlYS0xODY2LTQ2YjEtOTBlMy1mZmZiMDg3ZWJmOWIvcmVzb3VyY2Vncm91cHMveGlhbmd5YW4tbXNpL3Byb3ZpZGVycy9NaWNyb3NvZnQuQ29tcHV0ZS92aXJ0dWFsTWFjaGluZXMveGlhbmd5YW4iLCJ4bXNfdGNkdCI6IjEyODkyNDE1NDcifQ.I5YkAGUt9XXVoIL65NsL_X2x8MFWE6GfonEHB3GLu-rhDjLogMduvoaEWrlMdvBwHxIosEN8oZTakbJIB0KbDsl61Ye7IO9gqvv07sP8lxTycCU9cJF8jcaSq8o-b843qdpuBmaIC9WeKfCfgnJCfsadYqaZ1a8XtwDVNJsZcWh0k-OU3-kepW8GilWb6TWyAwpD0Xt7aGJlo243CoW3r2D8URk9Cyuma2dhzf88VXlKwUdRAVk8cprtXMGYe_YWlWtbW75NLrAZ1BjzB3hoZ2zm3u-_Sp4PXcMh5OB7qw9leVtX_jaenJQVbZPWnx_ABQeolAOJQUI0N9Gqzid2KA",
```

Comment:
It's probably not necessary to record tokens, so we should add a body key sanitizer in `conftest.py` to sanitize `access_token` fields in recordings -- here's an example of how that works: https://github.com/Azure/azure-sdk-for-python/blob/827028d04cd2e9380ba4a45e29ab80b7277387bd/sdk/advisor/azure-mgmt-advisor/tests/conftest.py#L51

If it's a hassle to re-record these tests, we can probably just manually replace the token values with `"access_token"` and verify with playback tests